.github/workflows/ci.yml

```diff
@@ -27,33 +27,45 @@ jobs:
       - uses: Swatinem/rust-cache@v2
 
       # Install OpenSSL Libraries for Windows
-      - name: install openssl
+      # - name: install openssl
+      #   if: matrix.os == 'windows-latest'
+      #   run: |
+      #     $ErrorActionPreference = "Stop"
+      #     choco install --verbose openssl
+      #     openssl version
+      #     refreshenv
+      #     echo "OPENSSL_LIB_DIR=C:/Program Files/OpenSSL-Win64/lib" >> $env:GITHUB_ENV
+      #     echo "OPENSSL_DIR=C:/Program Files/OpenSSL-Win64/" >> $env:GITHUB_ENV
+      #     echo "OPENSSL_INCLUDE_DIR=C:/Program Files/OpenSSL-Win64/include" >> $env:GITHUB_ENV
+      # Only test Rustls on Windows instead due to #1191
+      - name: "Interim Hacky Windows Test for #1191"
         if: matrix.os == 'windows-latest'
         run: |
```

Comment:
nit: probably a question about the workflows in general but should a [`shell`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) field be specified here? Action linters tend to enforce this at least in my experience.

Reply:
that's a good thing to be aware of at least. i wasn't.
wouldn't have expected it to matter here since it's not using any shell specific behaviour and just calling sed + cargo on string params, but maybe it was the cause of the earlier sed failure (maybe another shell than i expected actually did string interpolation first instead of passing the arg straight to sed).
anyway, i'll keep it in mind for any future improvements.

Reply:
it does actually look like `shell: bash` fixes the previous iteration (with the more precise sed). so using that. thanks!

Reply:
actually, i had to back out of it because for some reason now a random test is failing in windows and it is somehow connected to this... not sure the reason, but i'm so tired of this pr.